blueliquiddesigns/gravity-forms-pdf-extended
# Update Translations for 6.11 and load earlier

## Description

Register language files with WP before doing startup checks and update strings for 6.11

## Testing instructions
1. Deactivate Gravity Forms on a test site
2. Set the site language to Deutsch
3. Navigate to Settings -> General in WP admin area
4. Verify error message is fully translated

## Checklist:

## Additional Comments
